config/kernel-fpu.m4

```diff
@@ -27,18 +27,32 @@ AC_DEFUN([ZFS_AC_KERNEL_FPU_HEADER], [
 	],[
 		AC_DEFINE(HAVE_KERNEL_FPU_API_HEADER, 1,
 		    [kernel has asm/fpu/api.h])
-		AC_MSG_RESULT(asm/fpu/api.h)
+
+		ZFS_LINUX_TRY_COMPILE([
+			#include <linux/module.h>
+			#include <asm/fpu/internal.h>
```

Comment:
Nit: Should be a tabs I guess, not four spaces.

Reply:
I'm not sure I follow, tabs are used here not spaces.

Reply:
Sorry, a github formatting issue then.  Line 32 is indented 4 spaces whereas line 24 above is indented 8 spaces as expected. So I assumed spaces were used.